src/pages/EditProfile/index.tsx

```diff
@@ -8,70 +8,82 @@ import userImg from '../../media/user-img.svg'
+            <Form.Control type="text" placeholder="Last name" />
+          </Col>
+        </Form.Group>
 
-            <Form.Group as={Row} controlId="formHorizontalLName">
-              <Col sm="10">
-                <Form.Control type="text" placeholder="Last name" />
-              </Col>
-            </Form.Group>
+        <Form.Group as={Row} controlId="formHorizontalCountry">
+          <Form.Label column sm="2">
+            Country
+          </Form.Label>
+          <Col sm="4">
+            <Form.Control type="text" placeholder="Country" />
+          </Col>
+        </Form.Group>
 
-            <Form.Group as={Row} controlId="formHorizontalCountry">
-              <Col sm="10">
-                <Form.Control type="text" placeholder="Country" />
-              </Col>
-            </Form.Group>
+        <Form.Group as={Row} controlId="formHorizontalCity">
+          <Form.Label column sm="2">
+            City
+          </Form.Label>
+          <Col sm="4">
+            <Form.Control type="text" placeholder="City" />
+          </Col>
+        </Form.Group>
 
-            <Form.Group as={Row} controlId="formHorizontalCity">
-              <Col sm="10">
-                <Form.Control type="text" placeholder="City" />
-              </Col>
-            </Form.Group>
+        <Form.Group as={Row}>
+          <Form.Label as="legend" column sm={2}>
+            Relocate?
+          </Form.Label>
+          <Col as={Row} sm={10} className="mt-2">
+            <Form.Check
+              type="radio"
+              label="Yes"
+              name="formVerticalRadios"
+              id="formVerticalRadios1"
+              checked
+              className="pr-4"
+            />
+            <Form.Check
+              type="radio"
+              label="No"
+              name="formVerticalRadios"
+              id="formVerticalRadios2"
+            />
+          </Col>
+        </Form.Group>
 
-            <Form.Group as={Row}>
-              <Form.Label as="legend" column className="pl-4">
-                Open to Relocate?
-              </Form.Label>
-              <Col as={Row} className="mt-2">
-                <Form.Check
-                  type="radio"
-                  label="Yes"
-                  name="formVerticalRadios"
-                  id="formVerticalRadios1"
-                  checked
-                  className="pr-4"
-                />
-                <Form.Check
-                  type="radio"
-                  label="No"
-                  name="formVerticalRadios"
-                  id="formVerticalRadios2"
-                />
-              </Col>
-            </Form.Group>
-          </div>
-          <div className="m-auto hide-sm">
-            <img src={userImg} alt="userImg-svg" />
-            <br />
-            <CustomButton text="Upload" className="my-2 py-2" />
-          </div>
-        </div>
         <Form.Label as="legend" column sm={2} className="ml-n3">
           Education
         </Form.Label>
```

Comment:
I think you did not accept the incoming changes in this file and This cleared everything I worked on today.  Kindly check if you can revert this file.